crates/copilot/src/copilot_completion_provider.rs

```diff
@@ -492,8 +492,8 @@ mod tests {
             assert_eq!(editor.display_text(cx), "one.copilot2\ntwo\nthree\n");
             assert_eq!(editor.text(cx), "one.co\ntwo\nthree\n");
 
-            // Tabbing when there is an active suggestion inserts it.
-            editor.tab(&Default::default(), cx);
+            // AcceptInlineCompletion when there is an active suggestion inserts it.
+            editor.accept_inline_completion(&Default::default(), cx);
```

Comment:
I had to change to `editor.accept_inline_completion` since tab does not hold the completion logic anymore